packages/appstore/src/components/main-title-bar/total-asset.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Text, Popover } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { isMobile } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import BalanceText from 'Components/elements/text/balance-text';
+import { useTotalAccountBalance, useCFDAccounts, usePlatformAccounts, useThrottle } from '@deriv/hooks';
+
+const TotalAsset = observer(() => {
+    const { traders_hub, client, common } = useStore();
+    const { selected_account_type, is_eu_user, no_CR_account, no_MF_account } = traders_hub;
+    const { default_currency } = client;
+    const { current_language } = common;
+    const { real: platform_real_accounts, demo: platform_demo_account } = usePlatformAccounts();
+    const { real: cfd_real_accounts, demo: cfd_demo_accounts } = useCFDAccounts();
+    const platform_real_balance = useTotalAccountBalance(platform_real_accounts);
+    const cfd_real_balance = useTotalAccountBalance(cfd_real_accounts);
+    const cfd_demo_balance = useTotalAccountBalance(cfd_demo_accounts);
+    const is_real = selected_account_type === 'real';
+    const real_total_balance = platform_real_balance.balance + cfd_real_balance.balance;
+    const demo_total_balance = (platform_demo_account?.balance || 0) + cfd_demo_balance.balance;
+    const total_asset = is_real
+        ? useThrottle(real_total_balance, 60 * 1000)
+        : useThrottle(demo_total_balance, 60 * 1000);
+
+    // if selected region is non-eu, check active cr accounts, if selected region is eu- check active mf accounts
+    const has_active_related_deriv_account = !((no_CR_account && !is_eu_user) || (no_MF_account && is_eu_user));
```

Comment:
Does this cover for DIEL users who can have both CR and MF accounts?

Reply:
this is just moved from `asset-summary`. So if it worked previously then it covers DIEL as well.